### ISSUE TITLE
A series of errors during execution, could you provide requirements list?

### ISSUE THREAD

**Issue by Parseargs** [nwpu-zxr/VadCLIP#37]
Could you provide requirements for your code?

**Comment by Parseargs** [comment]
I am now facing a series of errors during training, including the AUC calculation error from SKlearn: 'Found input variables with inconsistent numbers of samples: [1109888, 11098880]' and XD dataset numpy reading problem: 'Cannot load file containing pickled data when allow_pickle=False'. I don't know whether these are caused by requirements or just data corruption.

**Comment by Parseargs** [comment]
Here is a requirement list that can perform a successful training process:
Brotli             1.2.0
certifi            2025.10.5
cffi               2.0.0
charset-normalizer 3.4.4
ftfy               6.3.1
h2                 4.3.0
hpack              4.1.0
hyperframe         6.1.0
idna               3.11
joblib             1.5.2
numpy              1.26.4
pandas             2.3.3
pillow             11.1.0
pip                25.2
pycparser          2.22
PySocks            1.7.1
python-dateutil    2.9.0.post0
pytz               2025.2
regex              2025.10.23
requests           2.32.5
scikit-learn       1.7.2
scipy              1.15.3
setuptools         80.9.0
six                1.17.0
threadpoolctl      3.6.0
torch              1.13.1
torchaudio         0.13.1
torchvision        0.14.1
tqdm               4.67.1
typing_extensions  4.15.0
tzdata             2025.2
urllib3            2.5.0
wcwidth            0.2.14
wheel              0.45.1
zstandard          0.25.0
All can successfuly perform training on python==3.10.0
